src/Integration.Vsix.UnitTests/CFamily/IntegrationTests/CLangAnalyzerTestFile_OneIssue_HasSecondaryLocations_response.json

```diff
@@ -0,0 +1,48 @@
+{
+  "Messages": [
+    {
+      "Filename": "",
+      "RuleKey": "S2259",
+      "PartsMakeFlow": true,
```

Comment:
FYI - We could use the property `PartsMakeFlow` to display the steps differently in the UI, should we want to.
For example, in cognitive complexity rule the property is False. In null reference it's True.

Reply:
Is this field passed to SonarQube/SonarCloud? (i.e. is it returned by the get issues API?)
If so then I think we should include it in the data model.

Reply:
No, it's only in CFamily objects. The API has "flows" property instead (that we used in the prototype).